kotlin/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
 
 tasks.test {
     useJUnit()
+    inputs.files(file("data/tests.json"))
```

Comment:
I don't know what this should be, but I had issues where kotlin wasn't rerunning when the json changed...

Reply:
@enzet happen to know?

Reply:
I will merge, to avoid being blocked. (but I still want to know how to fix this) :)
I never liked the gradle DSL...